vmdb/app/models/mixins/has_many_orchestration_stack_mixin.rb

```diff
@@ -7,8 +7,8 @@ module HasManyOrchestrationStackMixin
              :dependent   => :destroy
 
     has_many :direct_orchestration_stacks,
+             -> { where(:orchestration_stacks => {:ancestry => nil}) },
```

Comment:
Does `-> { where(:ancestry => nil) }` not work?

I had fixed this locally with the above, and don't believe I'm seeing a test failure to suggest it doesn't.